restapi/api.go

```diff
@@ -23,9 +25,20 @@ func Router(db *gorm.DB) (http.Handler, error) {
 		logger:   zap.S(),
 	}
 
-	rest.GetHandler(r, "/health", api.health)
-	rest.GetHandler(r, "/stories", api.ReturnAllStories)
-	rest.GetHandler(r, "/story/{storyID}", api.ReturnStoryByID)
+	// Public API
+	r.Group(func(r chi.Router) {
+		rest.GetHandler(r, "/health", api.health)
+		rest.GetHandler(r, "/login/{username}/{password}", api.Login)
```

Comment:
I don't like how password is a part of URL when making GET request for `/login`. Can we make this a POST endpoint and pass username and password as a part of header?